app/scaffold-controllers/searchBarController.js

```diff
@@ -0,0 +1,97 @@
+define([
+    'bluebird',
+    'plugins/webViewPlugin'
+],
+function(
+    Promise,
+    WebViewPlugin
+) {
+    var SearchBarController = function(webView, layout, searchConfig) {
+        this.viewPlugin = webView;
+        this.layout = layout;
+        this.searchConfig = searchConfig;
+
+        // Track the state to show.hide keyboard on toggle
+        //
+        // Set to true because it will be hidden when added
+        // to the parent layout
+        this.isVisible = true;
```

Comment:
Maybe it's just me, but could you explain the comment to me?

It says that it it is set to `true` by default because it will be hidden when added to parent layout, but the property name is called `isVisible`.. should setting `isVisible` to `true` correspond with the search bar being shown?

Perhaps its a typo in the comment.